### PR TITLE
Fetch active alarms from alarm_log

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
Fetch active alarms from the alarm log

##### Component Name
ACLK

### CODE REVIEW

File: health/health_json.c [hunk 1]
```diff
@@ -341,3 +341,30 @@ void health_alarms_values2json(RRDHOST *host, BUFFER *wb, int all) {
     buffer_strcat(wb, "\n\t}\n}\n");
     rrdhost_unlock(host);
 }
+
+
+void health_active_log_alarms_2json(RRDHOST *host, BUFFER *wb) {
+    netdata_rwlock_rdlock(&host->health_log.alarm_log_rwlock);
+
+    buffer_sprintf(wb, "{\n\t\"hostname\": \"%s\","
+                       "\n\t\"alarms\": {\n",
```
Comment (amoss) [inline health/health_json.c:350]: This one here should be [

**Comment by thiagoftsm** [comment]
During the compilation I detected this error:

```
aclk/agent_cloud_link.c: In function 'aclk_send_alarm_metadata':
aclk/agent_cloud_link.c:1660:5: warning: implicit declaration of function 'health_active_log_alarms_2json' [-Wimplicit-function-declaration]
 1660 |     health_active_log_alarms_2json(localhost, local_buffer);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by thiagoftsm** [comment]
Everything is working fine, after to fix the warning, this PR can be merged.
Thank you @stelfrag !

**Comment by amoss** [comment]
@thiagoftsm is right about the warning - @stelfrag add the prototype.

File: aclk/agent_cloud_link.c [hunk 2]
```diff
@@ -1645,17 +1646,18 @@ void aclk_send_alarm_metadata()
...
 
-    buffer_sprintf(local_buffer, ",\n\t \"alarm-log\" : ");
-    health_alarm_log2json(localhost, local_buffer, 0);
-    debug(D_ACLK, "Metadata %s with alarm_log has %zu bytes", msg_id, local_buffer->len);
 
-    buffer_sprintf(local_buffer, ",\n\t \"alarms-active\" : ");
-    health_alarms_values2json(localhost, local_buffer, 0);
-    debug(D_ACLK, "Metadata %s with alarms_active has %zu bytes", msg_id, local_buffer->len);
 
     buffer_sprintf(local_buffer, "\n}\n}");
```
Comment (gobwas) [inline aclk/agent_cloud_link.c:1662]: Do we still need to have two `}` here?

Comment (amoss) [reply aclk/agent_cloud_link.c:1662]: It's fine we have already tested it

**Pull request merged**